assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -151,49 +172,51 @@ function CurrentColorPicker({ rgb, hsl, hsv, hex, onChange, showOpacity }) {
   return (
     <Container>
       <Body showOpacity={showOpacity}>
-        <SaturationWrapper>
-          <Saturation
-            radius="8px"
-            pointer={() => (
-              <Pointer offsetX={-12} offsetY={-12} currentColor={rgb} />
-            )}
-            hsl={hsl}
-            hsv={hsv}
-            onChange={onChange}
-          />
-        </SaturationWrapper>
-        <HueWrapper>
-          <Hue
-            direction="horizontal"
-            height={`${CONTROLS_HEIGHT}px`}
-            radius={`${CONTROLS_BORDER_RADIUS}px`}
-            pointer={() => (
-              <Pointer offsetX={-12} offsetY={1} currentColor={rgb} />
-            )}
-            hsl={hsl}
-            onChange={onChange}
-          />
-        </HueWrapper>
-        {showOpacity && (
-          <AlphaWrapper>
-            <Alpha
+        <Suspense fallback={null}>
```

Comment:
I think we want a fallback of the correct size in here, as otherwise the popup is placed before the container has the right size (and thus dips below the viewport) - and it also somehow gets a horizontal scrollbar?

Note that I have the connection speed throttled to "Slow 3G" to force this.

![slow-react-color](https://user-images.githubusercontent.com/637548/125837537-fb24aca0-73ea-4ff0-84df-c85788e1b7f7.gif)

Reply:
Added:
![Screenshot 2021-07-16 at 11 34 59](https://user-images.githubusercontent.com/3294597/125918577-bfad1a44-2663-4f5e-a5e0-c24d3ee2da6b.png)